diagnostic_feedback/tests/integration/base_test.py

```diff
@@ -0,0 +1,166 @@
+from selenium.webdriver.support.ui import WebDriverWait
+from xblock.fields import String
+from xblockutils.base_test import SeleniumXBlockTest
+from xblockutils.resources import ResourceLoader
+# Studio adds a url_name property to each XBlock but Workbench doesn't.
+# Since we rely on it, we need to mock url_name support so it can be set via XML and
+# accessed like a normal field.
+from diagnostic_feedback.quiz import QuizBlock
+
+QuizBlock.url_name = String()
+
+loader = ResourceLoader(__name__)
+
+
+class DiagnosticFeedbackBaseTest(SeleniumXBlockTest):
+    """
+    Base class for integration tests.
+    Scenarios can be loaded and edited on the fly.
+    """
+
+    def _get_previous_button(self):
+        return self.browser.find_elements_by_xpath("//ul[@role='menu']/li")[0]
+
+    def _get_previous_button_link(self):
+        return self.browser.find_element_by_xpath("//ul[@role='menu']/li/a[@href='#previous']")
+
+    def _get_next_button(self):
+        return self.browser.find_elements_by_xpath("//ul[@role='menu']/li")[1]
+
+    def _get_next_button_link(self):
+        return self.browser.find_element_by_xpath("//ul[@role='menu']/li/a[@href='#next']")
+
+    def wait_until_has_class_disabled(self, elem):
+        """ Need to wait until the given element has disabled class """
+
+        wait = WebDriverWait(elem, self.timeout)
+        wait.until(lambda e: 'disabled' in elem.get_attribute('class'),
+                   u"{} should be disabled".format(elem.text))
```

Comment:
@jareerahsan Nit: As per the [Python guidelines](https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines), this should be formatted as follows:

```python
        wait.until(
            lambda e: 'disabled' in elem.get_attribute('class'),
            u"{} should be disabled".format(elem.text)
        )
```